src/assets/RustAsset.js

```diff
@@ -0,0 +1,93 @@
+const url = require('url');
+const path = require('path');
+const toml = require('toml');
+const {exec} = require('child-process-promise');
+
+const fs = require('../utils/fs');
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+
+const rustTarget = `wasm32-unknown-emscripten`;
```

Comment:
I think maybe we'll want to use wasm32-unknown-unknown, which doesn't require emscripten. Maybe checkout [rustify](https://github.com/browserify/rustify) which does something similar for browserify, and [wasm-experiments](https://github.com/killercup/wasm-experiments) to see how it all works.

Reply:
right now I'm using wargo and for now, wargo just uses wasm32-unknown-emscripten 

but sure, I'm going to take a look at rustify